pkg/installmanager/installmanager.go

```diff
@@ -513,6 +516,18 @@ func (m *InstallManager) generateAssets(provision *hivev1.ClusterProvision) erro
 		}
 	}
 
+	if src := m.ManifestsMountPath; isDirNonEmpty(src) {
+		m.log.Info("copying user-provided manifests")
+		dest := filepath.Join(m.WorkDir, "manifests")
+		out, err := exec.Command("bash", "-c", fmt.Sprintf("cp %s %s", filepath.Join(src, "*"), dest)).CombinedOutput()
```

Comment:
Slight different approach to the other places where we copy in installmanager, any reason to prefer one vs the other?

Reply:
I did not see any other places in installmanager where all files in a directory are copied. Can you point out where that is done?

Reply:
https://github.com/openshift/hive/blob/v1/pkg/installmanager/installmanager.go#L237

Reply:
That copies a single file with a well-known name. In the case of manifests, the installmanager does not know the names of the files to copy. It wants to copy all of the files in the directory.